### ISSUE TITLE
[a11y] Headings - Courses

### ISSUE THREAD

**Issue by tdy** [cs-education/classTranscribe#61]
Courses URL: https://classtranscribe.ncsa.illinois.edu/courses

Headings should be properly nested. This supports assistive technologies in organizing a page’s content for users who can’t see/scan the page’s layout (see [WCAG 2.0 SC 1.3.1 Info and Relationships][1]).

Currently the `<h5>` elements (Filter by Terms, Filter by Subject) aren’t properly nested. (Actually these probably shouldn't be headings at all from a semantic standpoint).

[1]: https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html
